fastkafka_gen/_code_generator/constants.py

```diff
@@ -31,3 +33,17 @@
         "output": 0.004
     },
 }
+
+# %% ../../nbs/Constants.ipynb 8
+INCOMPLETE_DESCRIPTION = "Please check if your application description is missing some crutial information:\n - Description of the messages which will be produced/consumed\n - At least one topic\n - At least one produces or consumes method\n"
```

Comment:
Remove this sentence "At least one produces or consumes method" and add something like "The business logic to implement while consuming/producing the messages"